ethereum/spec/src/main/resources/tech/pegasys/teku/spec/config/configs/mainnet.yaml

```diff
@@ -141,4 +141,7 @@ MAX_REQUEST_BLOB_SIDECARS: 768
 # `2**12` (= 4096 epochs, ~18 days)
 MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS: 4096
 # `6`
-BLOB_SIDECAR_SUBNET_COUNT: 6
\ No newline at end of file
+BLOB_SIDECAR_SUBNET_COUNT: 6
+
+# [New in Electra:EIP7251]
+MIN_PER_EPOCH_CHURN_LIMIT_ELECTRA: 128000000000 # 2**7 * 10**9 (= 128,000,000,000)
```

Comment:
Same as above, are we missing `MAX_PER_EPOCH_ACTIVATION_EXIT_CHURN_LIMIT`? (also applies to minimal.yaml)